### PR TITLE
Refatoring to remove unused code

### PR DESCRIPTION
**Pull request by StanleyGoldman**
There is a lot of unused code in our views.
As I'm copying chunks of the `SettingsView` and making changes to the `Window`, it makes sense to try to trim it back some.

### CODE REVIEW

**Comment by StanleyGoldman** [comment]
#252 Depends on this

**Comment by StanleyGoldman** [comment]
#261 Depends on this

File: src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs [hunk 1]
```diff
@@ -792,44 +517,5 @@ private void OnLoggingSettingsGui()
...
             EditorGUI.EndDisabledGroup();
         }
-
-        private void ResetInitDirectory()
-        {
-            initDirectory = Utility.UnityProjectPath;
-            GUIUtility.keyboardControl = GUIUtility.hotControl = 0;
-        }
-
-        private void ForceUnlockFile(object obj)
```
Comment (evilshana) [inline src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs]: We should make sure we track this in an issue for fixing, we'll need to do this as part of the initial file locking support.

Comment (StanleyGoldman) [reply src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs]: Who are you?

**Pull request merged**